packages/react-components/react-toast/src/components/Toaster.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * ⚠️ This is temporary and WILL be removed
+ */
+
+import * as React from 'react';
+import { Portal } from '@fluentui/react-portal';
+import { useToaster, getPositionStyles } from '../state';
+import { Toast } from './Toast';
+import { makeStyles, mergeClasses } from '@griffel/react';
+
+const useStyles = makeStyles({
+  container: {
+    position: 'fixed',
+  },
+});
+
+export const Toaster: React.FC = () => {
+  const { getToastToRender, isToastVisible } = useToaster();
+
+  const styles = useStyles();
+
+  return (
+    <Portal>
+      <div>
+        {getToastToRender((position, toasts) => {
+          return (
+            <div key={position} style={getPositionStyles(position)} className={mergeClasses(styles.container)}>
+              {toasts.map(({ content, ...toastProps }) => {
+                return (
+                  <Toast
```

Comment:
So in this case there's no way for the user to change the toast base component?! is it always `Toast`, shouldn't we provide a render prop for the user to decide what to render or something?

Reply:
This is just a sample implementation for now, the whole component hasn't been implemented yet. The base toast component will not be public though. Users will have a layout component.

The intention is definitely to provide users with a way of customizing this component even if it's not public tho